bugbear.py

```diff
@@ -12,7 +12,7 @@
 import attr
 import pycodestyle
 
-__version__ = "21.11.29"
+__version__ = "21.12.0"
```

Comment:
Lets not do this here and I'll do it on a diff when I release (as it's going to be 22.*)

Reply:
addressed